backends/backends.c

```diff
@@ -547,6 +584,30 @@ void *backends_main(void *ptr) {
             backend_set_kinesis_variables(&default_port,&backend_response_checker,&backend_request_formatter);
             break;
         }
+        case BACKEND_TYPE_MONGODB: {
+#if HAVE_MONGOC
+            if(unlikely(read_mongodb_conf(netdata_configured_user_config_dir,
+                                          &mongodb_uri,
+                                          &mongodb_database,
+                                          &mongodb_collection))) {
+                error("BACKEND: mongodb backend type is set but cannot read its configuration from %s/mongodb.conf",
+                      netdata_configured_user_config_dir);
+                goto cleanup;
+            }
+
+            if(likely(!mongodb_init(mongodb_uri, mongodb_database, mongodb_collection, mongodb_socket_timeout))) {
+                backend_set_mongodb_variables(&default_port, &backend_response_checker, &backend_request_formatter);
+                do_mongodb = 1;
+            }
+            else {
+                error("BACKEND: cannot initialize MongoDB backend");
+                goto cleanup;
+            }
+#else
+            error("BACKEND: MongoDB support isn't compiled");
```

Comment:
error or just warning/info?

Reply:
It's an error because it will be displayed when the user set
```
[backend]
	enabled = yes
	type = mongodb
```